clearnode/app_session_service.go

```diff
@@ -39,6 +39,7 @@ func (s *AppSessionService) CreateApplication(params *CreateAppSessionParams, rp
 	}
 	appSessionID := crypto.Keccak256Hash(appBytes).Hex()
 
+	participants := make(map[string]bool)
```

Comment:
Why is this a mapping and not an array. The for-loop below does not seem to have `continue` statements, which would advocate for the usage of the former.